### PR TITLE
Inertia switch

### PR DESCRIPTION
**Pull request by aehoppe**
Added section for inertia switch, even though it's identical to last year. It's not very fleshed out, but apparently it was good enough. Doesn't seem like it's that high-priority.

### CODE REVIEW

File: esf_main.tex [hunk 1]
```diff
@@ -271,12 +271,11 @@ \subsubsection{Description (type, operation parameters)}
...
 
 \subsubsection{Wiring/cables/connectors/}
-%Describe wiring, show schematics, describe connectors and cables used and show useful data regarding the wiring.
-TODO
+            The Inertia switch will be wired to be normally closed and open the shutdown circuit in the case that there is a crash. The inertia switch is wired in-line with the shutdown circuit to be normally closed. Please see \fref{fig:shutdown_circuit} for the position of the inertia switch relative to the other shutdown system components.
 
 \subsubsection{Position in car}
-%Provide CAD-renderings showing the relevant parts. Mark the parts in the rendering, if necessary.
-TODO
+ The inertia switch will be located on the dashboard, in reach of the driver as to fit EV5.7.4. It is mounted properly (upwards), so it can be triggered and reset.
```
Comment (byronwasti) [inline esf_main.tex]: Are we not going to include a render here? I'm pretty sure it is required; I would put a `TODO` here so we remember to.

File: esf_main.tex [hunk 2]
```diff
@@ -271,12 +271,11 @@ \subsubsection{Description (type, operation parameters)}
 	\end{table}
 
 \subsubsection{Wiring/cables/connectors/}
-%Describe wiring, show schematics, describe connectors and cables used and show useful data regarding the wiring.
-TODO
+            The Inertia switch will be wired to be normally closed and open the shutdown circuit in the case that there is a crash. The inertia switch is wired in-line with the shutdown circuit to be normally closed. Please see \fref{fig:shutdown_circuit} for the position of the inertia switch relative to the other shutdown system components.
```
Comment (byronwasti) [inline esf_main.tex]: This feels a bit wordy/repetitive. Possibly remove the part in the first sentence about the Inertia Switch being normally closed, and just leave that for the second sentence.

**Review by byronwasti — CHANGES_REQUESTED** [review]
Looks good. A render is needed, but for now a TODO is valid.

File: esf_main.tex (hunk 1 above)
Comment (aehoppe) [reply esf_main.tex]: So this is exactly what we had last year, which I guess passed. I'll still put in a render.

**Comment by aehoppe** [comment]
I added the inertia switch render.

**Comment by byronwasti** [comment]
You might want to add a distinct circle around the switch, as of right now there is a lot going on and its hard to tell what to look at.

**Comment by aehoppe** [comment]
Should be pretty visible. I added a yellow box around it.

**Review by byronwasti — APPROVED** [review]
